src/storage/googledrive.py

```diff
@@ -11,11 +11,21 @@
 from oauth2client import tools
 import logging
 from tools.config import CONFIG
+from tools.persistence import KVStore
 
 SCOPES = 'https://www.googleapis.com/auth/drive'
 CLIENT_SECRET_FILE = 'client_secret.json'
 APPLICATION_NAME = 'Instantshare'
 
+_name = "googledrive"
+# TODO: encryption
+kvstore = KVStore(_name)
+
+# FIXME: There is several problems with this:
```

Comment:
`googledrive.py` needs to be updated before it can make use of the new feature.
As it stands right now, it is very outdated, as explained in the code comments.